openapi.yaml

```diff
@@ -7848,6 +8489,7 @@ components:
         expires_at:
           description: The Unix timestamp (in seconds) for when the run will expire.
           type: integer
+          nullable: true
```

Comment:
Ah, nice to see that they fixed this. I had added this to the TypeSpec with a TODO explaining that it was missing from OpenAI's swagger spec. We can remove that TODO now. It's in `RunObject` in runs/models.tsp.